modules/monitoring-configuring-persistent-storage.adoc

```diff
@@ -20,4 +20,11 @@ Running cluster monitoring with persistent storage means that your metrics are s
 [NOTE]
 ====
 If you use a local volume for persistent storage, do not use a raw block volume, which is described with `volumeMode: Block` in the `LocalVolume` object. Prometheus cannot use raw block volumes.
+====
++
+[IMPORTANT]
+====
+For local persistent storage, Prometheus does not support file systems that are not POSIX compliant.
```

Comment:
I'm not sure about the "**local** persistent storage". Maybe just remove it?

Reply:
Removed.